arithmetization/src/main/java/net/consensys/linea/zktracer/module/ecdata/EcData.java

```diff
@@ -106,10 +106,53 @@ public void callEcData(
       case PRC_ECMUL -> ecMulEffectiveCall.addPrecompileLimit(1);
       case PRC_ECRECOVER -> ecRecoverEffectiveCall.addPrecompileLimit(1);
```

Comment:
Shouldn't these be conditional to stuff (at the very least: `INTERNAL_CHECKS_PASSED = true` ?